tests/test_xsd.py

```diff
@@ -7,10 +7,10 @@
 # ======================================================================================================================
 from __future__ import absolute_import
 from lxml import etree
+from pytest_zigzag import MK8S_ENV_VARS, ASC_ENV_VARS
+from zigzag.xml_parsing_facade import XmlParsingFacade
```

Comment:
Nitpick: PyCharm doesn't like this import because of reasons. Would you mind add the `# noinspection PyPackageRequirements` directive above this statement to shut it up?

Reply:
Don't see the change.